src/coreclr/vm/fieldmarshaler.h

```diff
@@ -92,7 +92,20 @@ class NativeFieldDescriptor
         return m_category;
     }
 
-    PTR_MethodTable GetNestedNativeMethodTable() const;
+    PTR_MethodTable GetNestedNativeMethodTable() const
```

Comment:
is this definition moved here for consistency?

Reply:
This is for resolving a build error.
```
  [ 93%] Building CXX object vm/wks/CMakeFiles/cee_wks_core.dir/__/weakreferencenative.cpp.o
  /usr/bin/riscv64-linux-gnu-ld: ../../dlls/mscordac/libmscordaccore.so: undefined reference to `NativeFieldDescriptor::GetNestedNativeMethodTable() const'
  clang: error: linker command failed with exit code 1 (use -v to see invocation)               
  make[2]: *** [debug/createdump/CMakeFiles/createdump.dir/build.make:308: debug/createdump/createdump] Error 1
```
fieldmarshalre.cpp is not included for createdump.
I resolved this error by putting the implemention into header.